conformance/tests/httproute-timeout-request.go

```diff
@@ -59,6 +59,10 @@ var HTTPRouteTimeoutRequest = suite.ConformanceTest{
 				Request:   http.Request{Path: "/request-timeout?delay=1s"},
 				Response:  http.Response{StatusCode: 504},
 				Namespace: ns,
+			}, {
+				Request:   http.Request{Path: "/disable-request-timeout?delay=2s"},
```

Comment:
Change this one to 1s too?

```suggestion
				Request:   http.Request{Path: "/disable-request-timeout?delay=1s"},
```

Reply:
oh whoops I changed the wrong one - do you want both of them to be 1s or just the new case I added?

Reply:
Both new tests with 1s delay are tested and working fine with Istio.